plugins/org.eclipse.reddeer.eclipse/src/org/eclipse/reddeer/eclipse/wst/server/ui/cnf/ServerSubmodule.java

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+ * Copyright (c) 2019 Red Hat, Inc and others.
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Red Hat, Inc - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.reddeer.eclipse.wst.server.ui.cnf;
+
+import org.eclipse.reddeer.swt.api.TreeItem;
+
+/**
+ * Represents a sub module assigned to a module.
+ * 
+ * @author zcervink@redhat.com
+ *
+ */
+public class ServerSubmodule {
+
```

Comment:
it would be also nice to have tests for it

Reply:
Tests added.